packages/config/src/bin/serve-chrome.ts

```diff
@@ -0,0 +1,156 @@
+import axios from 'axios';
+import path from 'path';
+import fs from 'fs';
+import { execSync, spawn } from 'child_process';
+import { load } from 'js-yaml';
+import { LogType, fecLogger } from '@redhat-cloud-services/frontend-components-config-utilities';
+import waitOn from 'wait-on';
+
+const REPO_OWNER = 'RedHatInsights';
+const REPO_NAME = 'insights-chrome';
+const CONTAINER_PORT = 8000;
+const CONTAINER_NAME = 'fec-chrome-local';
+const IMAGE_REPO = 'quay.io/cloudservices/insights-chrome-frontend';
+
+type ContainerRuntime = 'docker' | 'podman';
+let execBin: ContainerRuntime | undefined = undefined;
+
+const chromeDeploymentConfig = {
+  repo: 'git@gitlab.cee.redhat.com:service/app-interface.git',
+  deployFile: 'data/services/insights/frontend-base/deploy.yml',
+  tarTarget: path.resolve(__dirname, 'chrome-deploy/'),
+};
+
+const checkoutCommand = `git archive --remote=${chromeDeploymentConfig.repo}  HEAD ${chromeDeploymentConfig.deployFile} |  tar xvf - -C ${chromeDeploymentConfig.tarTarget}`;
+
+function checkContainerRuntime(): ContainerRuntime {
+  try {
+    if (execSync('which podman').toString().trim().length > 0) {
```

Comment:
Not blocking - but in the future it would be nice to make this configurable (versus trying podman first, then docker)

Reply:
FWIW Docker seemed considerably faster to me on Mac so I'd prefer this sooner than later :)

Reply:
@florkbr We should be preferring `podman`. and TBH, I wouldn't even add `docker` support.

Reply:
Yeah, we should be favoriting podman.